src/stories/components/Branding/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Branding
+
+.branding {
+  font-size: $font-size-xl;
+  font-weight: bold;
+  font-family: $header-font-family;
+}
+
+.logo {
+  margin-right: 5px;
```

Comment:
I decided to use 'pixels' here instead of 'rem' for the separation between the logo and the brand name because I thought the branding should remain consistent throughout the website. But if you think 'rem' is more appropriate here I can easily change it using the function provided by Alessia.

Reply:
Not really sure about this, but I remember we talked about using rem as it is more reliable than px, so I assume it is better to use margin-right: rem(5px);

Reply:
I think this okay for now.